controllers/usersignup/status.go

```diff
@@ -182,6 +182,43 @@ func (u *StatusUpdater) setStatusDeactivated(userSignup *toolchainv1alpha1.UserS
 		})
 }
 
+const (
+	UserMigrationFailed toolchainv1alpha1.ConditionType = "UserMigrationFailed"
```

Comment:
I should have commented that as the first thing - the condition should have more states than just "failed" ones, so we can fully monitor the process.